### DISCUSSION TITLE
docs: add a "cookbooks" section?

### DISCUSSION THREAD

**Discussion by caarlos0** [goreleaser/goreleaser#1541] [Q&A] [answered]
I was thinking about adding a "cookbooks" section of sorts with examples of how to do common things, for instance:

- compile multiple binaries
- release a library 
- upload binaries instead of archives
- etc

what do you folks think?

**Comment by cristaloleg** [comment]
That would be awesome <3

**Comment by caarlos0** [comment, marked as answer]
Its live! 

https://goreleaser.com/cookbooks
